Release 9.0.5 (#91)
Instabug/instabug-cordova: README.md

For more info, visit [Instabug.com](https://instabug.com).

### Installation

Currently, this plugin can only be installed via the Command-Line Interface.

```
cordova plugin add instabug-cordova
```

##### Ionic

If you're using this plugin with Ionic, you can install using this command.

```
ionic cordova plugin add instabug-cordova
```

## Usage

To initialize Instabug in your app, you need to do the following:

### Android

1. Change the name of the application class in your AndroidManifest.xml file to `android:name="com.instabug.cordova.plugin.MyApplication"`.

> Since `Cordova Android 7.0.0` AndroidManifest.xml position has changed and it is placed under `app/src/main/AndroidManifest.xml` ([read more](http://cordova.apache.org/announcements/2017/12/04/cordova-android-7.0.0.html)). You have to point it in your `.gradle` file in `platforms\android\com.instabug.cordova.plugin` directory in this way:

Reviewer: ```suggestion
Starting from _Cordova Android v7.0.0_ `AndroidManifest.xml` location has been changed and its new path now is `app/src/main/AndroidManifest.xml` ([read more](http://cordova.apache.org/announcements/2017/12/04/cordova-android-7.0.0.html)).

If you're using _Cordova Android v7+, please make sure you update its location inside Cordova's `build.gradle` file at `platforms/android/CordovaLib/build.gradle`:
```

```

> sourceSets {

Reviewer: ```suggestion
    sourceSets {
```

        main {
            manifest.srcFile 'src/main/AndroidManifest.xml'
        }
    }
```

> 2. You need to add your app token in the **MyApplication** class, by replacing `YOUR_ANDROID_TOKEN`. (You can find this class under this path `YourProjectName/platforms/android/app/src/main/java/com.instabug.cordova.plugin/MyApplication.java`)

Reviewer: It seems the path has been changed to `plugins/instabug-cordova/src/android/MyApplication.java` and therefore no integration code was added at my end.

Author: You will find it in the path mentioned in the Readme too.

3. You can change the invocation event by changing this line `InstabugInvocationEvent.SHAKE` in the **MyApplication** class to any of the following:

`InstabugInvocationEvent.FLOATING_BUTTON`, `InstabugInvocationEvent.SCREENSHOT_GESTURE`, `InstabugInvocationEvent.TWO_FINGER_SWIPE_LEFT`, or `InstabugInvocationEvent.NONE`.

4.  Make sure the following snippet is added to your project level `build.gradle`, if not you can manually add it as follows:.

```dart
allprojects {
	repositories {
	    maven {
	        url "https://sdks.instabug.com/nexus/repository/instabug-cp"
	    }
	}

> }

Reviewer: ```suggestion
    allprojects {
	repositories {
	    maven {
	        url "https://sdks.instabug.com/nexus/repository/instabug-cp"
	    }
	}
    }
```

    }
);
```

You can change the invocation event with any of the following: `'button'`, `'screenshot'`, `'swipe'`, or `'shake'`.

##### Ionic

> If you're using this plugin with Ionic, you need to add this line at the beginning of your **app.component.ts** class.

Reviewer: ```suggestion
##### TypeScript	
If you're using this plugin with TypeScript, you need to add this line at the beginning of your __app.component.ts__ class.
```
